lib/generators/clearance/routes/templates/routes.rb

```diff
@@ -1,12 +1,12 @@
-resources :passwords, controller: "clearance/passwords", only: [:create, :new]
-resource :session, controller: "clearance/sessions", only: [:create]
+  resources :passwords, controller: "clearance/passwords", only: [:create, :new]
+  resource :session, controller: "clearance/sessions", only: [:create]
 
-resources :users, controller: "clearance/users", only: [:create] do
-  resource :password,
-    controller: "clearance/passwords",
-    only: [:create, :edit, :update]
-end
+  resources :users, controller: "clearance/users", only: [:create] do
+    resource :password,
+      controller: "clearance/passwords",
```

Comment:
I prefer the way this reads in our routes file. Recent changes to our guides make this rule obsolete.